src/blockdata/script.rs

```diff
@@ -315,12 +315,12 @@ impl Script {
     }
 
     /// Generates P2WPKH-type of scriptPubkey
-    pub fn new_v0_wpkh(pubkey_hash: &WPubkeyHash) -> Script {
+    pub fn new_v0_p2wpkh(pubkey_hash: &WPubkeyHash) -> Script {
```

Comment:
In 234d9a49aec9a272857e5a4148d4cb5a2f1a6186
nit: requesting to add a deprecating change here since it is changing the function name